src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // string.split.filter => 1 object[key, value]
+  const styles = {};
+  const splitedString = sourceString.split(';').filter(Boolean);
+
+  for (const result of splitedString) {
```

Comment:
You can try to use reduce. In this case, you don't need initialize styles on 10 line (optional)